docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
+a configurable frequency and the output of the command becomes the label value.
 
-Dynamic labels, also known as **label commands**, allow you to generate labels
-at runtime. The `teleport` daemon will execute an external command on its host
-at a configurable frequency and the output of the command becomes the label's
-value.
+You can add multiple static and dynamic labels for the same resource.
+However, if you add static labels that use the same key but with different values, only the first 
+key-value pair is used.
 
-This is especially useful for decoupling the label's value from your Teleport
-configuration. For example, if you start Teleport on an Amazon EC2 instance, you
-can set a `region` label based on a command that sends a request to the EC2
-[instance metadata API](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-categories.html).
-This means that you can keep your configuration the same for each Node (and set
-the configuration within an Amazon Machine Image) while enabling users to search for Nodes by AWS
-region.
+Dynamic labels are especially useful for decoupling a label value from the Teleport configuration. 
+For example, if you start Teleport on an Amazon EC2 instance, you can use a dynamic label to set the 
+`region` value based on the result from a command sent to the EC2 instance 
+[metadata API](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-categories.html).
+The dynamic label enables you to use the same configuration for each server in an Amazon Machine Image 
+but filter and limit access to the servers based on their AWS region.
 
 ## Prerequisites
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-- (!docs/pages/includes/tctl.mdx!)
-- A host where you will run a Teleport Node. In this guide, we will apply labels
-  to this Node.
+(!docs/pages/includes/tctl.mdx!)
 
-<Admonition title="Resources you can label" type="tip">
+## Step 1/3. Install Teleport
 
-You can label any resource managed by Teleport using the instructions in this
-guide. See the following guides for how to add resources to a Teleport cluster:
+1. Select a host for running the Teleport agent.
 
-|Guide|Notes on labeling|
-|---|---|
-|[Applications](../../application-access/getting-started.mdx)||
-|[Databases](../../database-access/getting-started.mdx)|Teleport can discover some AWS-hosted databases automatically. In this case, labels will also be automatically applied. |
-|[Kubernetes clusters](../../kubernetes-access/getting-started.mdx)||
-|[Servers](../../server-access/getting-started.mdx)||
-|[Windows desktops](../../desktop-access/getting-started.mdx)|Dynamic labels are not supported for Windows desktops. See our [Desktop Access documentation](../../desktop-access/rbac.mdx#labeling) for how to label desktops.|
+1. (!docs/pages/includes/install-linux.mdx!)
 
-</Admonition>
+1. Generate an invitation token for your local computer.
```

Comment:
Again, I would recommend we don't say "local computer."

In the first step of this section we said "select a host" so the user could have selected any host.